docs/explanation/pairing-flow-architecture.md

```diff
@@ -180,7 +180,6 @@ When the user accepts the confirmation screen on the Desktop device, it will use
 - Check that client_id is the expected value; error out if not.
 - Check that state exists and is well formed; error out if not.
 - Build the scoped-key bundle for `"https://identity.mozilla.com/apps/oldsync"`, encrypting it with the provided `keys_jwk` to produce `keys_jwe`.
-- Generate a BrowserID assertion with the fxa-oauth-server as audience.
```

Comment:
Interesting, I haven't tested pairing flow but this seems to imply it still uses it? I'll try and verify.

Reply:
@clouserw I wasn't able to test the pairing flow locally but did search in [Firefox code](https://searchfox.org/mozilla-central/search?q=browserid&path=&case=false&regexp=false). Nothing came up about pairing, I think this is good to go 👍🏽